docs/source/en/add_new_model.md

```diff
@@ -889,3 +889,68 @@ used by hundreds and possibly even thousands of developers and researchers. You
 your achievements with the community.
 
 **You have made another model that is super easy to access for everyone in the community! 🤯**
+
+## Model additions and their timeline: when is a model added to transformers?
+
+We aim for `transformers` to have support for new model architectures and checkpoints as early as possible:
+availability can range from day-0 (and hour-0) releases for some models, to a few days/weeks for others.
+
+The availability of this is usually up to the model contributors, as well as how excited the community is for the
+architecture.
+
+We can split the model architecture possibilities in four sections:
+- Day-0 integration
+- Same-week integration
+- Post-release integration
+- Hub-first release
+
+Let's dive into each of these and see how we (the transformers team) can help you contribute your architecture and get
+your architecture to be very simply used by all members of the community.
```

Comment:
Does "very simply" here refer to how easy it is to use the model, or that adding in the model to transformers only requires enabling the simplest of feature integrations to be mergeable? It reads to me as the latter, but I'm assuming the former. If so, I'd maybe go for "easily" instead of "simply"

Reply:
Ah yeah, it's the former! Thanks!